MiniApp/Classes/RealMiniApp.swift

```diff
@@ -48,6 +48,13 @@ internal class RealMiniApp {
                 }
                 self.downloadMiniApp(appInfo: appInfo, completionHandler: completionHandler, messageInterface: messageInterface)
             case .failure(let error):
+                let createError = error as NSError
```

Comment:
This block is duplicate code to `MiniAppDownloader.handleDownloadError`, isn't it? Maybe you could move `handleDownloadError` to this class and then you could handle it in the `.failure` case of `downloadMiniApp`?

Reply:
We can do that, in that case, we need to call self.downloadMiniApp() again in .failure() flow

Reply:
I fixed those changes now